src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -1046,7 +1047,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             If type.IsErrorType() OrElse
                SourceModule IsNot type.ContainingModule OrElse
-               type.TypeKind <> TYPEKIND.Class Then
+               type.TypeKind <> TypeKind.Class Then
```

Comment:
Most of the changes in this file can be reverted.

Reply:
They were made automatically by the IDE. The casing is incorrect so any edit in this file will make the IDE normalize the casings and I see no point in trying to keep the casing unchanged.

Reply:
I understand it will be annoying, but please do it anyway. Reverting PRs is made more complex when there are a large number of changes unrelated to the change in a file.

Reply:
OK.